app/src/main/java/com/example/paritosh/stormy/WeatherDataProvider.java

```diff
@@ -55,6 +58,30 @@ public void onResponse(@NonNull Call call, Response response) throws IOException
 
     }
 
+    private ArrayList<Hourly> hourlyDetails(String jsonData) throws JSONException {
```

Comment:
this method should be incorporated in getCurrentDetails and also rename it to getCurrentWeatherDetails